proto/skaffold.proto

```diff
@@ -104,9 +104,24 @@ message Event {
         ResourceStatusCheckEvent resourceStatusCheckEvent = 6; // indicates progress for each kubernetes deployment.
         FileSyncEvent fileSyncEvent = 7; // describes the sync status.
         DebuggingContainerEvent debuggingContainerEvent = 8; // describes the appearance or disappearance of a debugging container
+        DevLoopEvent devLoopEvent = 9; // describes a start and end of a dev loop.
     }
 }
 
+// `DevLoopEvent` marks the start and end of a dev loop.
+message DevLoopEvent {
+    int32 iteration = 1; // dev loop iteration. 0 represents initialization loop.
+    string status = 2; // dev loop status oneof: In Progress, Completed, Failed
+    ErrDef err = 3; // actionable error message
+
+}
+
+// `ErrDef` defines an error occurred along with an optional suggestions
```

Comment:
No backticks

Reply:
All the protos struct defs have backticks in the description hence added these.